app/upgrades/v19_2/constants.go

```diff
@@ -1,11 +0,0 @@
-// Copyright Tharsis Labs Ltd.(Evmos)
-// SPDX-License-Identifier:ENCL-1.0(https://github.com/evmos/evmos/blob/main/LICENSE)
-
-package v192
```

Comment:
did you check if removing the last upgrade, the e2e test still works? does it pick the 'latest' docker img tag? or it does not run at all?